testsuite/shared/src/main/java/org/jboss/as/test/integration/security/common/servlets/SimpleServletWithScope.java

```diff
@@ -0,0 +1,79 @@
+package org.jboss.as.test.integration.security.common.servlets;
```

Comment:
Missing copyright header.

Reply:
Since this new file is specifically for OIDC tests, we can add it in the `testsuite/integration/elytron-oidc-client/subsystem` directory instead.